site/admission/index.html

```diff
@@ -134,14 +134,48 @@ <h2>Application Process</h2>
 </body>
 
 <body>
-  <footer>Audition Requirements</footer>
-  <div>Art & Design</div>
-  <div>Dance</div>
-  <div>Fashion Technology</div>
-  <div>Music</div>
-  <div>Dance</div>
-  <h3>FAQ</h3>
-  <p>TBD</p>
+  <div class="audition-requirement">
+    <h3 class="audition-requirement-title">Audition Requirements</h3>
+    <div class="accordian-box">
+      <details class="accordian">
+        <summary class="accordian-label">Art & Design</summary>
+        <div class="accordian-inner">
+          Something small enough to escape casual notice.
+        </div>
+      </details>
+      <details class="accordian">
+        <summary class="accordian-label">Dance</summary>
+        <div class="accordian-inner">
+          Something small enough to escape casual notice.
+        </div>
+      </details>
+      <details class="accordian">
+        <summary class="accordian-label">Fashoin Technology</summary>
+        <div class="accordian-inner">
+          Something small enough to escape casual notice.
+        </div>
+      </details>
+      <details class="accordian">
+        <summary class="accordian-label">Music</summary>
+        <div class="accordian-inner">
+          Something small enough to escape casual notice.
+        </div>
+      </details>
+      <details class="accordian">
+        <summary class="accordian-label">Dance</summary>
+        <div class="accordian-inner">
+          Something small enough to escape casual notice.
+        </div>
+      </details>
+    </div>
+  </div>
+</body>
```

Comment:
Watch out here, you are closing the `body`. This is most likely gonna cause stuff to break. Probably got added by accident. Delete this closing body tag and the opening one below it.